framework/captcha/drivers/DriverFactory.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace yii\captcha\drivers;
+
+use Yii;
+use yii\base\Object;
+use yii\base\InvalidConfigException;
+
+/**
+ *
+ */
+class DriverFactory extends Object
```

Comment:
This class seems to be redundant: we do not introduce 'factories' classes over the framework especially while DI is used.
Functionality of this class should be moved backed to `CaptchaAction`.